source/onboarding/person/index.html.md.erb

```diff
@@ -10,12 +10,22 @@ weight: 2
 Join the following 
 
 - AAD groups - submit pull request to the [group management repository](https://github.com/hmcts/ansible-management/blob/master/azure-aad-vars/main.yml)
-- Github - join the appropriate team group
-- Office 365 team
 
-Add [Github Slack user mapping](https://github.com/hmcts/github-slack-user-mappings)
+```yaml
+- name: username
+  groups:
+  - dcd_group_allusers_v2
+  - dcd_reform_dev_logs
+  - dcd_reform_sandbox_azure
+  - dcd_reform_monitoring_reader
```

Comment:
It's expected that all developers are in the below group too, it gives access to non production vaults, and the ability to request access to the 'SC' group
```suggestion
  - dcd_reform_monitoring_reader
  - DTS CFT Developers
```